packages/js/src/generators/library/files/lib/README.md

```diff
@@ -14,6 +14,6 @@ Run `<%= cliCommand %> build <%= name %>` to build the library.
 
 ## Running unit tests
 
-Run `<%= cliCommand %> test <%= name %>` to execute the unit tests via [Jest](https://jestjs.io).
+Run `<%= cliCommand %> test <%= name %>` to execute the unit tests via <%= if(options.unitTestRunner === 'jest') { %> [Jest](https://jestjs.io) <%= } else { %> [Vitest](https://vitest.dev/) <% } %>.
```

Comment:
I think you need to remove the `=` before the `if`. 

Here's how the syntax should be:
```
    <% if(MY_CONDITION) { %>
    WRITE_THIS <% } else { %>
    WRITE_THIS
  <% } %>
```